ex1/functions.py

```diff
@@ -65,23 +80,53 @@ def f():
 # compare against. The function should return a list with equal length as the
 # input list containing for each number in the original list either True or
 # False if the number was greater than or equal to n.
+def check_list(l,num):
+    for i, item in enumerate(l):
+        if l[i] >= num:
+            l[i] = True
```

Comment:
This can be dangerous since it changes the original list and fills it with `False` and `True`